Dockerfile

```diff
@@ -29,4 +29,25 @@ ENV FLASK_ENV=development
 ENV FLASK_DEBUG=1 
 
 ENTRYPOINT ["poetry"]
-CMD ["run", "flask", "run", "--host=0.0.0.0"]
\ No newline at end of file
+CMD ["run", "flask", "run", "--host=0.0.0.0"]
+
+FROM base as test
+
+RUN apt-get update -qqy && apt-get install -qqy wget gnupg unzip
+# Install Chrome
+RUN wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | apt-key add - \
+  && echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google-chrome.list \
+  && apt-get update -qqy \
+  && apt-get -qqy install google-chrome-stable \
+  && rm /etc/apt/sources.list.d/google-chrome.list \
+  && rm -rf /var/lib/apt/lists/* /var/cache/apt/*
+# Install Chrome WebDriver
+RUN CHROME_MAJOR_VERSION=$(google-chrome --version | sed -E "s/.* ([0-9]+)(\.[0-9]+){3}.*/\1/") \
+  && CHROME_DRIVER_VERSION=$(wget --no-verbose -O - "https://chromedriver.storage.googleapis.com/LATEST_RELEASE_${CHROME_MAJOR_VERSION}") \
+  && echo "Using chromedriver version: "$CHROME_DRIVER_VERSION \
+  && wget --no-verbose -O /tmp/chromedriver_linux64.zip https://chromedriver.storage.googleapis.com/$CHROME_DRIVER_VERSION/chromedriver_linux64.zip \
+  && unzip /tmp/chromedriver_linux64.zip -d /usr/bin \
+  && rm /tmp/chromedriver_linux64.zip \
+  && chmod 755 /usr/bin/chromedriver
+
+ENTRYPOINT ["poetry", "run", "pytest"]
```

Comment:
We should add some documentation to your Readme for how to run these tests inside Docker now that that's an option